model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -203,34 +206,33 @@ def make_model_config(
         model_config_dict["name"] = variant_name
```

Comment:
I can't say I fully understand the changes. Should this still be here?

Reply:
Yes, until we make the switch to use the API we are still using the variant name in both the output repo and as the model name in the ModelConfig.